src/__tests__/integration/scenarios/sci-e.ts

```diff
@@ -4,9 +4,9 @@ import {
   npmInstallPackage,
   npmUninstallPackage,
 } from '../helpers/module-installer';
-import {execPromise, getJSONFromText} from '../helpers/common';
+import {execPromise} from '../helpers/common';
 import {sciEInputData} from '../test-data/sci-e';
-
+import * as YAML from 'js-yaml';
```

Comment:
please import libs from the beginning of the file, and then newline, then other internal dependencies